docs/docs/cmd/aad/app/app-open.md

```diff
@@ -0,0 +1,42 @@
+# aad app open
+
+Gets the url of the Azure AD app registration page to be able to quickly navigate to the Azure Portal. 
+
+## Usage
+
+```sh
+m365 aad app open [options]
+```
+
+## Options
+
+`--appId <appId>`
+: Application (client) ID of the Azure AD application registration to get.
```

Comment:
`...to open`, as we're not retrieving the information about the AAD app using this command